autoware_launch/config/control/vehicle_cmd_gate/vehicle_cmd_gate.param.yaml

```
@@ -3,7 +3,9 @@
     update_rate: 10.0
     system_emergency_heartbeat_timeout: 0.5
     use_emergency_handling: false
+    check_external_emergency_heartbeat: false
     use_start_request: false
+    enable_cmd_limit_filter: false
```

Please make it true by defalut
```suggestion
    enable_cmd_limit_filter: true
```

---

@TakaHoribe done :+1:.

---

Thanks!